OpenRA.Game/Sound/Sound.cs

```diff
@@ -81,9 +83,20 @@ ISoundSource LoadSound(ISoundLoader[] loaders, IReadOnlyFileSystem fileSystem, s
 
 		public void Initialize(ISoundLoader[] loaders, IReadOnlyFileSystem fileSystem)
 		{
+			StopMusic();
+			soundEngine.ReleaseSourcePool();
+
+			if (sounds != null)
+				foreach (var soundSource in sounds.Values)
+					soundSource.Dispose();
+
+			GC.Collect();
```

Comment:
We do a `GC.Collect` at the end of map and mod loading anyway, so one here isn't needed.